package.json

```
@@ -4,8 +4,15 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "node index"
+    "start": "nodemon --inspect index"
```

```suggestion
    "start": "nodemon index",
    "debug": "nodemon --inspect index"
```
위와 같이 스타트와 디버그를 분리하는건 어떨까요?